packages/workbox-core/_default.mjs

```diff
@@ -49,12 +49,16 @@ class WorkboxCore {
     if (process.env.NODE_ENV !== 'production') {
       const padding = '   ';
       logger.groupCollapsed('Welcome to Workbox!');
+      if (self.location.hostname === 'localhost') {
+        logger.unprefixed.log(`The development build with extra logging is ` +
+          `automatically enabled when accessing your web app via localhost.`);
+      }
```

Comment:
I'm fine with the above but it does only cater to one situation where we set dev build on localhost.

We could change this up a little to be more of a statement

> You are using the development build of Workbox. Learn more how to this works here

Where here is either a link to a docs page or the message is a group and we add info on:
- The localhost vs non localhost builds
> The setConfig options to force a build type
> Setting the log level for additional logs

Reply:
https://developers.google.com/web/tools/workbox/guides/troubleshoot-and-debug would probably be that doc—I can link to it in this log message, and then we can follow-up on modifying the information there in a PR against WebFundamentals at some point.